src/Config.ts

```diff
@@ -0,0 +1,3 @@
+export const version = '2.5.5';
+export const testing = false;
+export const lastUpdated: Date = new Date('##LAST_UPDATED##');
```

Comment:
```suggestion
export const lastUpdated = new Date('##LAST_UPDATED##');
```
we don't need this extraneous cast